apps/functions-subscription/src/domain/__tests__/data.ts

```diff
@@ -12,7 +12,7 @@ const aSubscriptionHistoryIdV1 =
   'aSubscriptionHistoryIdV1' as SubscriptionHistoryId;
 const aSubscriptionId = 'aSubscriptionId' as SubscriptionId;
 const aUserId = 'aUserId' as UserId;
-const aTrialId = 'aTrialId' as TrialId;
+const aTrialId = 'aTrialId012345678901234567' as TrialId;
```

Comment:
**question**: why is this necessary?

are you sure we need to update this?
IMHO this is not needed in the domain level

Reply:
Reverted in eff99bbaf41355bc1a98ee131ab9a4d2694136f4